standard/expressions.md

```diff
@@ -71,11 +71,16 @@ Static binding takes place at compile-time, whereas dynamic binding takes place
 > *Example*: The following illustrates the notions of static and dynamic binding and of binding-time:
 >
 > ```csharp
-> object o = 5;
-> dynamic d = 5;
-> Console.WriteLine(5); // static binding to Console.WriteLine(int)
-> Console.WriteLine(o); // static binding to Console.WriteLine(object)
-> Console.WriteLine(d); // dynamic binding to Console.WriteLine(int)
+> using System;
```

Comment:
I'm not sure whether this change actually makes it more readable.

We can definitely create a template that has some common using directives and then puts the code from the example into a method in a library. I think I'd prefer that over adding the method declaration explicitly in this example, but I'm willing to be persuaded.

Reply:
removed using directive